templates/navigation-submenu.twig

```diff
@@ -0,0 +1,25 @@
+<nav class='nav-submenu'>
+	<div class='nav-submenu-wrapper'>
```

Comment:
Have you tried your suggestion on your local ?

Basically It's used since we have an 8px distance between the navigation bar and the subitems's nav, then you will be able to "hover" on the empty space between each other.